internal/flag/flags.go

```diff
@@ -148,4 +148,8 @@ const (
 	SubscriptionID                  = "subscriptionId"                  // SubscriptionID flag
 	ResourceGroup                   = "resourceGroup"                   // ResourceGroup flag
 	VNet                            = "vnet"                            // VNet flag
+	AccountID                       = "accountId"                       // AccountID flag
+	ContainerID                     = "containerID"                     // ContainerID flag
```

Comment:
we don't follow go rules for flags
```suggestion
	ContainerID                     = "containerId"                     // ContainerID flag
```

Reply:
also we may not need this flag at all

Reply:
I agree with you, in the doc is listed as Required but it is not true 🤷‍♂️

Reply:
it is but ware being sneaky about it, there's this rule where there can only be one container per region per provider, so what we are doing is checking if it exists if not we create it and use that id